### ISSUE TITLE
[Bug]: query is none after load data successfully

### ISSUE THREAD

**Issue by aressu1985** [matrixorigin/matrixone#3141]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):
- Hardware parameters:0.5.0
- OS type:
- Others:
```

### Actual Behavior

mysql> load data infile '/home/sudong/mo-tpch/data/lineitem.tbl' into table TPCH.lineitem FIELDS TERMINATED BY '|' LINES TERMINATED BY '\n';
Query OK, 60175 rows affected (0.61 sec)
Records: 60175  Deleted: 0  Skipped: 0  Warnings: 0 WriteTimeout: 0

mysql> select count(*) from lineitem;
+----------+
| count(*) |
+----------+
|        0 |
+----------+
1 row in set (0.00 sec)

mysql>

**Comment by iamlinjunhong** [comment]
select count() from lineitem is syntax error. Why would you execute successfully? Can you give a clear context? @aressu1985

**Comment by iamlinjunhong** [comment]
fix in pr #3278

**Comment by aressu1985** [comment]
fixed
